### PR TITLE
Add resource limits

### PR DESCRIPTION
**Pull request by SzyWilliam**
Add resource limits

### CODE REVIEW

**Comment by SzyWilliam** [comment]
image

File: helm/values.yaml [hunk 1]
```diff
@@ -19,12 +19,18 @@ datanode:
   nodeCount: 3
   dataDir: /iotdb/data
   storageCapacity: 10Gi
+  resourceLimits:
```
Comment (xingtanzjr) [inline helm/values.yaml:22]: change to `resource` and make `limits` and `requests` as its sub-variable

File: docker/Dockerfile_from_git [hunk 2]
```diff
@@ -1,23 +0,0 @@
-# NOTICE: for distribution, we shall not include maven/git
```
[the remark this answers is not in the document]
Comment (SzyWilliam) [reply docker/Dockerfile_from_git]: Done

File: helm/templates/confignode.yaml [hunk 3]
```diff
@@ -46,6 +46,10 @@ spec:
           mountPath: {{ .Values.confignode.dataDir }}
         - name: config
           mountPath: /iotdb/conf_tmp
+        resources:
+          limits:
```
[the remark this answers is not in the document]
Comment (SzyWilliam) [reply helm/templates/confignode.yaml]: Done

File: helm/values.yaml (hunk 1 above)
Comment (SzyWilliam) [reply helm/values.yaml]: Done

**Review by xingtanzjr — APPROVED** [review]

**Pull request merged**